examples/platforms/posix/flash-windows-stubs.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
@chshu, This looks fine. I will have to implement this in the Windows driver to add real support. I don't think there is any real worth adding this to the ot-cli for Windows.